doc/usage/extensions/coverage.rst

```diff
@@ -22,6 +22,14 @@ should check:
 
 .. confval:: coverage_ignore_classes
 
+.. confval:: coverage_ignore_py
+
+   List of `Python regular expressions`_.
+
+   If any of these regular expressions matches any part of the full import path
+   of a Python API member, that Python API member is excluded from the
```

Comment:
"Python API member" is common wording?

Reply:
Switched to ‘Python objects’. I had no idea what the common wording was :)